src/integration-test/java/se/sundsvall/document/apptest/DocumentIT.java

```diff
@@ -25,6 +26,7 @@
 import se.sundsvall.dept44.test.annotation.wiremock.WireMockAppTestSuite;
 import se.sundsvall.document.Application;
 
+@Disabled // TODO: Remove before flight
```

Comment:
It might be just me but Ive stared at this todo for five minutes and cant for the life figure out what it supposed to mean.

Reply:
It means that the test is disabled

Reply:
Ahh... the text.

https://www.naa.edu/remove-before-flight/

(I.e. we need to remove this annotation, before story-completion)

Reply:
Ok I assume activating it is part of another ticket?

Reply:
Yes, removing the annotation will make the IT-tests fail (in their current state).
All tests must be reworked (and activated). This will happen in UF-6329.